src/libcharon/plugins/kernel_super/oper_db_client/liboper_db_client/liboper_db_client.h

```diff
@@ -14,10 +14,13 @@ typedef struct liboper_db_obj_t liboper_db_obj_t;
 
 enum LIBOPER_DB_CLIENT_CLASS_IDS {
     LIBOPER_DB_CLIENT_CLASS_ID_TERMINATOR,
+    LIBOPER_DB_CLIENT_VRF_TUNNEL_COUNT_ID_TERMINATOR
```

Comment:
All labels in this enum should begin with `LIBOPER_DB_CLIENT_CLASS_ID`
```suggestion
    LIBOPER_DB_CLIENT_CLASS_ID_VRF_TUNNEL_COUNT_ID_TERMINATOR
```

Also I think it should end with **IN**_TERMINATOR